com.unity.render-pipelines.universal/Runtime/UniversalRenderer.cs

```diff
@@ -572,7 +574,13 @@ public override void Setup(ScriptableRenderContext context, ref RenderingData re
 #endif
 
             if (this.actualRenderingMode == RenderingMode.Deferred)
+            {
+                var cmd = CommandBufferPool.Get();
+                CoreUtils.SetKeyword(cmd, ShaderKeywordStrings.RenderPassEnabled, useRenderPassEnabled && cameraData.cameraType == CameraType.Game);
+                context.ExecuteCommandBuffer(cmd);
+                cmd.Release();
```

Comment:
Ah I see how it's used now, I'm not sure why the other one is necessary though, you could just enable once per-camera. Also there's a small overhead to execute command buffer, I'd recommend combining this execution of other command buffers we already have.

Reply:
These two calls are required for the runtime fallback in case something changes with the settings, the previous call being - illegal command is found and RenderPass option is turned off in a hard manner (shouldn't be reached unless something goes drastically wrong, f.e. Blit in a RenderPass is found and etc.). I'll check whether it falls back properly when enabled in per-camera manner as it's possible this is a bit of an overkill. And I also agree about the cmd buffer, will try to find a better place to squeeze this in tomorrow